server/src/migrations/20250109000000-report-companies.ts

```diff
@@ -0,0 +1,17 @@
+import { getDbCollection } from "@/common/utils/mongodbUtils"
+
+export const up = async () => {
+  await getDbCollection("reported_companies").updateMany(
+    {},
+    {
+      $set: {
+        reason: "vide (migration)",
```

Comment:
```suggestion
        reason: "migration",
```
plus simple à filtrer sur MB